src/patternfly/components/Switch/styles.scss

```diff
@@ -0,0 +1,129 @@
+@import "../../patternfly-utilities";
+
+.pf-c-switch {
+  --pf-c-switch--Height: calc(#{var(--pf-global--FontSize--md)} * #{var(--pf-global--LineHeight--md)});
+  --pf-c-switch--Width: calc(#{var(--pf-c-switch--Height)} * #{var(--pf-global--golden-ratio)});
+  --pf-c-switch--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-switch__toggle--BorderRadius: var(--pf-c-switch--Height);
+  --pf-c-switch__toggle--Padding: calc(#{var(--pf-c-switch--Height)} / 2 * (1 - 1 / #{var(--pf-global--golden-ratio)}));
+  --pf-c-switch__toggle--BackgroundColor: var(--pf-global--primary-color--100);
+  --pf-c-switch__toggle--m-off--BackgroundColor: var(--pf-global--BackgroundColor--dark-200);
+  --pf-c-switch__toggle--m-disabled--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-200);
+  --pf-c-switch__toggle--before--Width: calc(#{var(--pf-c-switch--Height)} / #{var(--pf-global--golden-ratio)});
+  --pf-c-switch__toggle--before--Height: var(--pf-c-switch__toggle--before--Width);
+  --pf-c-switch__toggle--before--BorderRadius: var(--pf-c-switch__toggle--before--Width);
+  --pf-c-switch__toggle--before--BoxShadow: var(--pf-global--BoxShadow--md);
+  --pf-c-switch__toggle--before--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-switch__toggle--before--m-disabled--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+  --pf-c-switch__toggle--before--RighPosition: calc(#{var(--pf-c-switch--Width)} - #{var(--pf-c-switch__toggle--before--Width)} - 2 * #{var(--pf-c-switch__toggle--Padding)});
+  --pf-c-switch__label--Color: var(--pf-global--Color--dark-100);
+  --pf-c-switch__label--m-checked--Color: var(--pf-global--disabled-color--100);
+  --pf-c-switch__label--m-disabled--Color: var(--pf-global--disabled-color--100);
+  --pf-c-switch__label--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-switch__label--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-switch__label--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-switch__label--m-on--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-switch__label--m-off--PaddingRight: var(--pf-global--spacer--md);
+
+  position: relative;
+  display: inline-block;
+  height: var(--pf-c-switch--Height);
+  line-height: var(--pf-c-switch--LineHeight);
+  vertical-align: middle;
+  cursor: pointer;
+
+  .pf-c-switch__input {
+    position: absolute;
+    cursor: pointer;
+    opacity: 0;
+
+    &:checked {
+      ~ .pf-c-switch__label.pf-m-off {
+        color: var(--pf-c-switch__label--m-checked--Color);
```

Comment:
`m-checked` isn't a modifier class. To follow BEM, I would probably change this `--pf-c-switch__input--checked__label--m-off--Color`